src/CMakeLists.txt

```diff
@@ -2,10 +2,18 @@
 
 # Create a list of the files to be preprocessed
 set(fppFiles
+    stdlib_32_bit_fnv_hashes.fypp
+    stdlib_32_bit_nmhashes.fypp 
+    stdlib_32_bit_water_hashes.fypp 
+    stdlib_64_bit_fnv_hashes.fypp 
+    stdlib_64_bit_pengy_hashes.fypp 
+    stdlib_64_bit_spookyv2_hashes.fypp 
```

Comment:
Should those be named consistently with the module/submodule?

Reply:
Do you mean as follows:
```
    stdlib_hash_32bit_fnv.fypp
    stdlib_hash_32bit_nm.fypp 
    stdlib_hash_32bit_water.fypp 
    stdlib_hash_64bit_fnv.fypp 
    stdlib_hash_64bit_pengy.fypp 
    stdlib_hash_64bit_spookyv2.fypp 
```

Reply:
Proposed in [this PR](https://github.com/wclodius2/stdlib/pull/11)